point/chain.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
+  "chain_id": "point_10687-1",
+  "bech32_prefix": "point",
+  "node_home": "$HOME/.pointd",
+  "genesis": {
+    "genesis_url": "https://github.com/pointnetwork/point-chain-config/blob/main/mainnet-1/genesis.json"
+  },
+  "daemon_name": "pointd",
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "apoint",
+        "low_gas_price": 5000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+        {
+            "denom": "apoint"
+        }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/point/point",
+    "recommended_version": "v7.0.0",
+    "compatible_versions": [
+      "v7.0.0"
+    ],
+    "binaries": {
+      "linux/amd64": "",
+      "linux/arm64": "",
+      "darwin/amd64": "",
+      "darwin/arm64": "",
+      "windows/amd64": ""
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "8673c1f04c29c464189e8bf29e51fb0b38da2f19",
+        "address": "rpc-mainnet-1.point.space:26656",
```

Comment:
@DaMandal0rian In your version you have different ports for seeds and peers 26656 & 26657. Should it be different? I have same as in config.toml